schema/url/pass/plugin-map-markers-overlay.raml

```diff
@@ -18,45 +18,48 @@ types:
         required: false
       xAxisVariable: 
         type: VariableSpec
-      # TODO rename to geoAggregateVariable after b57
-      facetVariable: 
-        type: VariableSpec[]
-        maxItems: 2
-        required: false
+      geoAggregateVariable: 
+        type: VariableSpec
```

Comment:
i dont follow - why did we change to VariableSpec (from VariableSpec[])?

Reply:
Should only take a single geohash at a time. It's a new type of strata var essentially. If we need facets in the map viz at some point well add them back then, but they're not currently spec'd.